BDPointiOSXamarinDemo/AppDelegate.cs

```diff
@@ -25,8 +25,11 @@ public override bool FinishedLaunching(UIApplication application, NSDictionary l
 
             // Override point for customization after application launch.
             // If not required for your application you can safely delete this method
-            BDLocationManager.Instance.SessionDelegate = new SessionDelegate(this);
-            BDLocationManager.Instance.LocationDelegate = new LocationDelegate(this);
+            BDLocationManager.Instance.GeoTriggeringEventDelegate = new GeoTriggeringEventDelegate(this);
+            BDLocationManager.Instance.TempoTrackingDelegate = new TempoTrackingDelegate(this);
+
+            UNUserNotificationCenter.Current.RequestAuthorization(UNAuthorizationOptions.Alert, (approved, err) => {});
```

Comment:
I think we need to either log an error, present an [alert view](https://docs.microsoft.com/en-us/xamarin/ios/user-interface/controls/alerts), or present a local notificaiton.

Reply:
Done. Please review again